server/src/db/seeders/ReviewSeeder.js

```diff
@@ -0,0 +1,54 @@
+import {Review, User, Product} from '../../models/index.js'
+
+class ReviewSeeder {
+  static async seed() {
+    const testUser = await User.query().findOne({email:'test@gmail.com'})
+    const testUserTwo = await User.query().findOne({email: 'testTwo@gmail.com'})
+    const artisanBread = await Product.query().findOne({productName:'Artisan Baker White Bread Gluten Free'})
+    const heritageBread = await Product.query().findOne({productName:'Heritage Style Whole Grain Bread Gluten Free' })
+    const chexCereal = await Product.query().findOne({productName:'Corn Cereal'})
+    const reviewsData = [
+      {
+        rating: 1,
+        title: 'Toast it to get the most of it',
+        content: 'This bread is unedible unless you toast it and cover it in butter.',
+        userId: testUser.id,
+        productId: artisanBread.id
+      },
+      {
+        rating: 5,
+        title: 'God tier bread',
+        content: 'Eating this bread was like bathing in the moon pools of Aphrodites palace.',
```

Comment:
yo dude....can I have some of this fine bread maybe? Ill merge all your PRs if you share